Kernel/Memory/AnonymousVMObject.h

```diff
@@ -83,11 +85,13 @@ class AnonymousVMObject final : public VMObject {
     };
 
     LockWeakPtr<AnonymousVMObject> m_cow_parent;
+    Spinlock m_shared_cow_lock { LockRank::None };
     LockRefPtr<SharedCommittedCowPages> m_shared_committed_cow_pages;
```

Comment:
Can we use SpinlockProtected here instead?

Reply:
Done.